testing/e2e/clients/aks.go

```diff
@@ -246,6 +246,14 @@ func NewAks(ctx context.Context, subscriptionId, resourceGroup, name, location s
 		clientId:       clientID,
 		options:        options,
 	}
+
+	// install gateway api (in place of managed gateway addon for now)
+	if err := cluster.runCommand(ctx, armcontainerservice.RunCommandRequest{
+		Command: to.Ptr("kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.2.0/standard-install.yaml"),
```

Comment:
why not use the local installation? safer, no?

Reply:
applying from github is similar to what users do. don't really want this to be the full gateway installation solution, just a temporary unblocker which this is the simplest solution for

I can switch if needed but would requiring zipping up then applying which is more work for something we'll shortly remove

Reply:
it's also not a huge risk since this is just for test infra

Reply:
ok yeah that's fine then